esmvalcore/cmor/_fixes/cmip6/fgoals_f3_l.py

```diff
@@ -0,0 +1,41 @@
+"""Fixes for CMIP6 FGOALS-f3-L."""
+import cftime
+import numpy as np
+
+from ..fix import Fix
+
+
+class AllVars(Fix):
+    """Fixes for all vars."""
```

Comment:
```suggestion
    """Fixes for all vars."""
    
```

Reply:
I am getting the same issue in virtually all pull requests due to the yapf precommit hook, which keeps removing this blank line. Any ideas how to resolve that?